README.md

```diff
@@ -217,9 +216,9 @@ The project has the following folders and files:
 
 ### Terraform plugins
 
-- [Terraform](https://www.terraform.io/downloads.html) 0.11.x
-- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) 1.20.X
-- [terraform-provider-google-beta](https://github.com/terraform-providers/terraform-provider-google-beta) 1.20.X
+- [Terraform](https://www.terraform.io/downloads.html) 0.12
```

Comment:
Let's add a note about the last 0.11.x-compatible version, like https://github.com/terraform-google-modules/terraform-google-network/pull/55

Reply:
Fixed in 639a3b7.